sc-tools/sc-server/src/sc-server-impl/sc_server.cpp

```diff
@@ -21,7 +21,7 @@ ScServer::ScServer(std::string hostName, size_t port)
 
   {
     LogMessage(ScServerErrorLevel::info, "Socket data:");
-    LogMessage(ScServerErrorLevel::info, "\tHost name: " + m_hostName);
+    LogMessage(ScServerErrorLevel::info, "\tHost: " + m_hostName);
```

Comment:
if message is changed to "Host", shouldn't m_hostName be changed to m_host as well?